charts/latest/blob-csi-driver/templates/csi-blob-node.yaml

```diff
@@ -33,7 +33,7 @@ spec:
 {{ toYaml .Values.imagePullSecrets | indent 8 }}
       {{- end }}
       hostNetwork: true
-      dnsPolicy: Default
+      dnsPolicy: ClusterFirstWithHostNet
```

Comment:
i believe this is needed because of the EDGE_CACHE_CONFIG_ENDPOINT.  If you parameterize that, is there a way for the appropriate service fqdn to be filled in by helm rather than changing the dnsPolicy?

Reply:
We could use params, but we will need this DNS policy regardless. Without it, the service is not resolvable even by fqdn.

Reply:
This is required by grpc communications internal to the edgecache implementation  then?

Reply:
This is required so that csi-blob-node can send `CreateBlob/GetBlob`.

The other communications to initiate mount with the cached pod is done thru a unix socket under /csi/nfsd_mounts.sock

Reply:
do you have more details of following config:
```
            - name: EDGE_CACHE_CONFIG_ENDPOINT
              value: hydrad-config:8675
```

We hit some issues in the before when set as `ClusterFirstWithHostNet` in AKS managed control plane, that's why we changed to `Default` in the end.

Reply:
When `iscacheenabled` is set for a volume, the node will mount the blob storage through the edge cache. In order for edge cache to provide a mount point, we must send a GRPC request to edge cache config service to configure the endpoint.

`hydrad-config` is the name (for now) of the config service for edge cache which is a clusterip service in kube.

When I left dnsPolicy: Default, I couldn't resolve this kube service, even by fqdn.

What issues did you see with ClusterFirst enabled?